scripts/skype

```diff
@@ -23,12 +23,19 @@ else
       fi
     fi
   done
+  video=""
+  for f in /dev/video*; do
+    test -c $f &&
+      video="$video --device=$f:$f:rwm"
+  done
+    
 
   echo "Starting ${prog}..."
-  docker run -d \
+  docker run --rm -d \
```

Comment:
@darashenka `--rm` is not compatible with `-d`, please remove the `--rm` i am pretty sure you did not plan on committing this change.